nusight2/src/client/components/localisation/view.tsx

```diff
@@ -360,10 +361,12 @@ const RobotComponents: React.FC<RobotRenderProps> = observer(({ robot, model })
         defaultColor="orange"
       />
 
-      {model.fieldIntersectionsVisible && robot.fieldIntersections && (
-        <FieldIntersections intersections={robot.fieldIntersections} />
+      {model.fieldIntersectionsVisible && robot.fieldIntersectionsF && (
+        <FieldIntersections intersections={robot.fieldIntersectionsF} />
```

Comment:
wheres the `F` come from

Reply:
It's to represent that they are in field space, I will update naming convention to match typical convention